aviary/docs/getting_started/installation.md

```diff
@@ -1,319 +1,277 @@
+
+### Installing with conda
+If you have a conda environment, then you can install pyOptSparse with access to the IPOPT optimizer. This is the most commonly used optimizer in Aviary, as it is generally more performant than SLSQP while also being free. Simply install `pyoptsparse` using conda:
 
 ```
-$ conda activate av1
+conda install pyoptsparse
 ```
 
-### Installing Additional Dependencies
+### Installing with pip, or if you have SNOPT
+To install pyOptSparse without conda, you will need the [build_pyoptsparse](https://github.com/OpenMDAO/build_pyoptsparse) package to help build pyOptSparse correctly. It is possible to install pyOptSparse without this utility package, but that is outside the scope of this guide. 
 
-Aviary can run in MPI.
-You do not need the `mpi4py` or `petsc4py` packages to use Aviary.
-They are useful when running large OpenMDAO models in parallel.
-If needed, you can install them with the following command:
+First install `build_pyoptsparse`, then run it as a command:
 
 ```
-$ conda install -c conda-forge mpi4py petsc4py
+pip install git+https://github.com/OpenMDAO/build_pyoptsparse
+build_pyoptsparse
 ```
 
-Download developer version of `OpenMDAO`:
-
-```
-$ cd ~/workspace
-$ git clone git@github.com:OpenMDAO/OpenMDAO.git
-```
+#### Adding SNOPT
+[SNOPT](http://ccom.ucsd.edu/~optimizers/solvers/snopt/) is a proprietary, high-performance optimizer that is very good at solving large nonlinear problems and is used by many OpenMDAO users. SNOPT is only supported on Linux.
 
-You have a new subdirectory `workspace/OpenMDAO`. You are not expected to modify source code of OpenMDAO, but you want to keep up with the latests version of it. The best way to do is to install OpenMDAO in developer mode. This removes the need to reinstall OpenMDAO after changes are made. Go to this directory where you see a file `setup.py`. Run
+If you have a copy of SNOPT, instead run the following command to install `pyoptsparse` with both IPOPT and SNOPT available:
 
 ```
-$ cd OpenMDAO
-$ pip install -e .
+build_pyoptsparse -s /path_to_SNOPT_dir
 ```
 
-You should see something like the following:
+### Troubleshooting
+There are several places this installation can fail, here are a couple troubleshooting steps to take. First, try running `build_pyoptsparse -v` to get a more verbose output.
 
+*If you see an error for a missing command:*
 ```
-Successfully installed networkx-3.1 openmdao-3.29.1.dev0
+ERROR: Required command swig not found.
 ```
+Then try and install the missing package (in this case `swig`) using pip or conda. Swig is a commonly missing dependency.
 
-Now, let us install `dymos` in a similar way:
-
+*If you see an error where a conda or mamba command failed:*
 ```
-$ cd ~/workspace/
-$ git clone git@github.com:OpenMDAO/dymos.git
-$ cd dymos
-$ pip install -e .
+subprocess.CalledProcessError: Command '['mamba', 'info', '--unsafe-channels']' returned non-zero exit status 109.
```

Comment:
We should try to collect our common installation errors (in all of our upcoming refreshes) to add to this database.